### ISSUE TITLE
A11y: Tooltip does not show for Share button on keyboard focus/activation

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/clean-and-green-philly#602]
## Describe the task
When you hover over the "Share" button with the mouse, a tooltip appears that tells the user they can copy the link. After clicking on the button, a tooltip tells the user that the action was successful. These tooltips should also be visible on keyboard focus.

## Acceptance Criteria

- [ ] Open a property page
- [ ] TAB over to the "Share" button.
- [ ] On keyboard focus, the "Copy link" tooltip should be visible.
- [ ] Press ENTER to activate the button.
- [ ] The "Link copied" tooltip should be visible.

## Additional context

* Possibly helpful: https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible and https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-within

**Comment by bacitracin** [comment]
Assigned!
